src/idom/html.py

```diff
@@ -280,7 +280,18 @@
 
 def _(*children: Any, key: Key | None = None) -> VdomDict:
     """An HTML fragment - this element will not appear in the DOM"""
-    return vdom("", *children, key=key)
+    attributes, coalesced_children = separate_attributes_and_children(children)
```

Comment:
Need to rename `coalesced_children` to `children`. We should do a global search on `coalesce` just to clean up references.